typing/src/main/java/org/navgurukul/typing/utils/Logger.java

```diff
@@ -0,0 +1,46 @@
+package org.navgurukul.typing.utils;
+
+import android.util.Log;
+
+public class Logger {
+    private static final String TAG = "TypingAssist";
+    private static final boolean DEBUG = true;
+    private static final boolean VLOG = DEBUG && true;
+    private static final boolean DLOG = DEBUG && true;
+    private static final boolean ILOG = DEBUG && true;
+    private static final boolean WLOG = DEBUG && true;
+    private static final boolean ELOG = DEBUG && true;
+
+    private Logger() {
```

Comment:
this class can be moved to common utils and can be used throughout the project. A method to print StackTrace can also be added here